Sources/ProjectDescription/Config.swift

```diff
@@ -34,23 +34,29 @@ public struct Config: Codable, Equatable {
     /// Cloud configuration.
     public let cloud: Cloud?
 
+    /// Cache configuration.
+    public let cache: Cache?
+
     /// Initializes the tuist configuration.
     ///
     /// - Parameters:
     ///   - compatibleXcodeVersions: List of Xcode versions the project is compatible with.
     ///   - cloud: Cloud configuration.
+    ///   - cache: Cache configuration.
     ///   - plugins: A list of plugins to extend Tuist.
     ///   - generationOptions: List of options to use when generating the project.
     public init(
         compatibleXcodeVersions: CompatibleXcodeVersions = .all,
         cloud: Cloud? = nil,
+        cache: Cache? = nil,
```

Comment:
I was wondering if we should provide the default value for the config at the project's manifest level 🤔 For now, the default value of a config is provided at the `TuistCore.Cache` level. 

Thoughts?

Reply:
This is the behavior that I'd expect as a user:

- Passed flavor exists: It uses it  and tuist prints the flavor
- Passed flavor doesn't exist: It fails the execution.
- No flavor is passed and there are flavors defined: It picks the one from the list, Tuist's output tells me that it's taking the default one.
- No flavor is passed and the list of flavors is empty: It defaults to Tuist's defaults and tells me what the defaults are.

Reply:
Awesome, I added unit tests to verify described scenarios 💪